### PR TITLE
libxmlb: 0.1.10 → 0.1.11

### PR DESCRIPTION
**Pull request by jtojnar**
https://github.com/hughsie/libxmlb/blob/0.1.11/NEWS

Also formatted the expression and added installed tests

### CODE REVIEW

File: pkgs/development/libraries/libxmlb/default.nix [hunk 1]
```diff
@@ -28,6 +68,12 @@ stdenv.mkDerivation rec {
 
   doCheck = true;
 
+  passthru = {
+    tests = {
+      installed-tests = nixosTests.libxmlb;
```
Comment (worldofpeace) [inline pkgs/development/libraries/libxmlb/default.nix]: Is the precedent for this not on hold anymore?

Comment (jtojnar) [reply pkgs/development/libraries/libxmlb/default.nix]: No idea, just wanted to try it out.

Comment (worldofpeace) [reply pkgs/development/libraries/libxmlb/default.nix]: Checked, guess it's not :+1:

**Review by worldofpeace — APPROVED** [review]
LGTM. Built the following
```
discover fwupd gnome3.gnome-software
```

**Pull request merged**